index.bs

```diff
@@ -4511,18 +4799,24 @@ partial dictionary MLOpSupportLimits {
     <tr>
       <th>operand</th>
       <th>[=/allowed data types=]</th>
+      <th>[=/minimum allowed data types=]</th>
       <th>[=/allowed ranks=]</th>
+      <th>[=/minimum allowed ranks=]</th>
     </tr>
   </thead>
   <tr>
     <td>{{input}}</td>
     <td>[=/any data type|any=]</td>
+    <td>{{MLOperandDataType/"float32"}}, {{MLOperandDataType/"float16"}}, {{MLOperandDataType/"int32"}}</td>
```

Comment:
🤔 It's interesting that `gather` takes uint8 in all the API's apparently, but `expand` does not? Request double check? I'd generally expect in ML API's that any pure data movement operators take consistent data types.